client/tink_keyset_helper.go

```diff
@@ -80,3 +81,43 @@ func convertTinkKeysetHandleToBytes(keysetHandle *keyset.Handle) ([]byte, error)
 	}
 	return bytesBuffer.Bytes(), nil
 }
+
+// addNewTinkKeyset receives a knox version list and a tink key templateFunc, create a new tink keyset contains
+// a single fresh key from the given tink key templateFunc. Most importantly, the ID of this signle fresh key is
+// different from the ID of all existed tink keys in the given knox version list (avoid Tink key ID duplications).
+func addNewTinkKeyset(templateFunc func() *tinkpb.KeyTemplate, knoxVersionList knox.KeyVersionList) ([]byte, error) {
+	existedTinkKeysID := make(map[uint32]struct{})
```

Comment:
nit: existed->existing

Reply:
Thank you for your comment! I revised the code accordingly.